platform/android/detect.py

```diff
@@ -119,7 +119,9 @@ def mySpawn(sh, escape, cmd, args, env):
 			env.extra_suffix=".armv7.neon"+env.extra_suffix
 		else:
 			env.extra_suffix=".armv7"+env.extra_suffix
-
+        elif env['android_arch']=='armv8':
```

Comment:
Sorry to bother you with rebasing again, but all your `elif`s have indentation issues (spaces instead of tabs).